docs/rendering/modelextensions/facedata.md

```diff
@@ -0,0 +1,74 @@
+Face Data
+=========
+
+Additional data about an element's faces can be specified at either the element level or the face level in a vanilla "elements" model and in generated item models loaded through the `forge:item_layers` model loader. Faces which do not specify their own face data will fall back to the element's face data or a default if no face data is specified at the element level. All values of the face data are optional.
```

Comment:
This is very misleading since it seems to talk about an extension to vanilla which then tells you to use Forge's model loader.

Reply:
Separated the mention of the Forge loader into a separate paragraph and elaborated on why that loader is needed